models/articles.model.js

```diff
@@ -80,3 +80,15 @@ ORDER BY comments.created_at DESC;
       return rows;
     });
 };
+
+exports.updateVotes = (inc_votes, article_id) => {
+  const queryString = `
+UPDATE articles
+SET    votes = votes            + $1
+WHERE  article_id = $2 returning *; 
+    `;
+  const values = [inc_votes, article_id];
+  return db.query(queryString, values).then(({ rows }) => {
+    return rows;
```

Comment:
take the object off the array here so that we are returning only one thing instead of an array with one item in which is unnecessary 👍 (your test will have to be updated first though 😄 )